src/test/scala/chiselTests/experimental/hierarchy/SeparateElaborationSpec.scala

```diff
@@ -0,0 +1,113 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chiselTests.experimental.hierarchy
+
+import chiselTests.ChiselFunSpec
+import chisel3._
+import chisel3.stage.{ChiselGeneratorAnnotation, ChiselStage, DesignAnnotation}
+import chisel3.internal.ImportedDefinitionAnnotation
+import chisel3.experimental.hierarchy.{Definition, Instance}
+import firrtl.options.TargetDirAnnotation
+
+import java.nio.file.Paths
+import scala.io.Source
+
+class SeparateElaborationSpec extends ChiselFunSpec with Utils {
+  import Examples._
+
+  describe("(0): Elaborating an Instance separately from its Definition") {
+    it("should result in an instantiation in FIRRTL without a module declaration.") {
+      val testDir = createTestDirectory(this.getClass.getSimpleName).toString
+      val dutAnnos = (new ChiselStage).run(Seq(
+        ChiselGeneratorAnnotation(() => new AddOne),
+        TargetDirAnnotation(testDir),
+      ))
+
+      // Grab DUT definition to pass into testbench
+      val designAnnos = dutAnnos.flatMap { a =>
```

Comment:
```suggestion
      val designDefs = dutAnnos.flatMap { a =>
```

isnt what is returned a Definition[AddOne]?